libs/template/presets/storefront/experience/pages/cms-pages.ts

```diff
@@ -0,0 +1,34 @@
+import { cmsTypes } from '@oryx-frontend/content';
+import { ExperienceComponent } from '@oryx-frontend/experience';
+
+export const cmsPages: ExperienceComponent[] = cmsTypes.map((type) => ({
```

Comment:
```suggestion
export const cmsPage: ExperienceComponent[] = cmsTypes.map((type) => ({
```

Reply:
But we define here multiple pages

Reply:
As for me we are defining a typical cms page here.

Reply:
Fixed

Reply:
Can you please also fix a file name?